kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -79,7 +79,7 @@
 }
 
 .kb-staging-table-header__name {
-    width: 27%;
+    width: 26%;
```

Comment:
had to bump down size to allow for padding increase in full container (see below line 303)